README.md

```diff
@@ -103,8 +103,15 @@ $ python manage.py startapp myapp --template https://github.com/mongodb-labs/dja
 
 ### Configuring the `DATABASES` setting
 
-After you've set up a project, configure Django's `DATABASES` setting similar
-to this:
+After you've set up a project, configure Django's `DATABASES` with
+`django_mongodb.parse(uri)`:
+
+```python
+MONGODB_URI = "mongodb://<my_user>:<my_password>@localhost:27017/my_database"
+DATABASES["default"] = django_mongodb.parse(MONGODB_URI)
+```
+
+Alternatively, you can configure the DATABASES settings directly:
```

Comment:
Explain why would I want or need to use the alternative. Add backticks around DATABASES.